handlers/validators.go

```diff
@@ -16,6 +16,79 @@ import (
 
 var validatorsTemplate = template.Must(template.New("validators").ParseFiles("templates/layout.html", "templates/validators.html"))
 
+type ValidatorDataQueryParams struct {
```

Comment:
Was not sure about this, maybe we should put this also into types-module? (helper-function inclusive)